nemo/collections/asr/parts/numba/rnnt_loss/rnnt_numpy.py

```diff
@@ -90,6 +90,25 @@ def _assert_no_grad(tensor):
     )
 
 
+class LogSoftmaxGradModification(Function):
```

Comment:
Can this class be used for any activation? In yes, how about giving it a more general name as I do not see anything specific to logsoftmax?

Reply:
Actually yeah you can use it to inject gradient update anywhere in the DAG but for now I want to keep this specific to LogSoftmax clamping because 1) it wastes memory - it copies the input joint tensor which is so expensive for rank 4 tensor 2) its forward backward is written specifically with clamping in mind. 

We can refactor this out (or remove it) if I can figure out how to do in place log softmax computation on CPU, but that will be a long term future task (if its even possible)..